### ISSUE TITLE
[BUG] Custom DnD 5e and TAH incompatible

### ISSUE THREAD

**Issue by ctbritt** [Larkinabout/fvtt-token-action-hud-dnd5e#113]
**Describe the bug**

When enabled, Custom DnD 5e blocks Token Action HUD from showing. 

**Module Data**
[Custom DND5E.json](https://github.com/user-attachments/files/17077775/Custom.DND5E.json)

**Steps to reproduce**
1. Enabled Custom DnD 5e, Token Action HUD dnd 5e, and Token Action Hud Library

**Versions:**
- Token Action Hud Core 1.5.7
- Token Action Hud Dnd5e 1.5.8
- Custom DnD 5e: 1.0.4
- System: dnd5e 3.3.1
- Foundry VTT: 12.331
- Browser: Chromium 129.0.6668.59

**Additional context**

**Error Message:**
![CleanShot 2024-09-20 at 12 05 17](https://github.com/user-attachments/assets/bde3f94f-04a1-44b9-89ed-f9438541997c)

**Comment by Larkinabout** [comment]
I think this may just be a Custom D&D 5e issue. It looks like one of the configs is missing the label. I'm not able to reproduce the issue. Try updating to Custom D&D 5e 1.0.5 and see if the issue is still happening. If so, click on underlined link in the the error at the very top and send me a screenshot of where the error is occuring in the code (underlined in red).

**Comment by ctbritt** [comment]
nope. still not working with 1.0.5. Here's the location of the error: 
![CleanShot 2024-09-25 at 00 20 46@2x](https://github.com/user-attachments/assets/affac307-5448-4b62-8538-5b10c2ec555a)

**Comment by ctbritt** [comment]
and here's the console error: 
![CleanShot 2024-09-25 at 00 21 52@2x](https://github.com/user-attachments/assets/d9e72820-c093-4e00-8700-91652c606bcd)

**Comment by Larkinabout** [comment]
Revert back to Token Action HUD D&D 5e 1.5.8 as 1.6.0 is for dnd5e v4 only.

**Comment by Darkfoundry** [comment]
I need manifests 
CORE 1.5.7 
d&d 1.5.8. 
Where do I find
